common-api/src/main/java/org/jboss/galleon/util/IoUtils.java

```diff
@@ -152,10 +152,26 @@ public static void copy(Path source, Path target) throws IOException {
     }
 
     public static void copy(Path source, Path target, boolean skipExistingFiles) throws IOException {
-        if(Files.isDirectory(source)) {
-            Files.createDirectories(target);
+        if (Files.isDirectory(source)) {
+            try {
+                Files.createDirectories(target);
+            } catch (FileAlreadyExistsException e) {
+                // FileAlreadyExistsException happen if the target path already exists and it's not directly (symbolic link or regular file)
```

Comment:
I don't fully follow the content of the comment. The sentence seems incomplete. From the Javadoc, we can read: "if dir exists but is not a directory". if yo ucould clarify in the comment. Thank-you.